### PR TITLE
 Add styles for product purchase order note

### PR DESCRIPTION
**Pull request by AlejandroCuriel**
- What?: Some styles were added to the ticket that is generated when a product is purchased.
 - Why?: These styles were added so that the user can better visualize the purchased product. See #70  for more information.
 
## Screenshots 
Screenshot from 2022-05-22 20-59-43

### CODE REVIEW

File: app/assets/stylesheets/components/_cards.scss [hunk 1]
```diff
@@ -91,3 +91,19 @@
...
+  border: 2px solid $black;
+  box-shadow: -4px 11px 32px -2px rgba(0,0,0,0.73);
+  height: 250px;
+  margin: 0 auto;
+  width: 700px;
+
+  h2 {
+    text-transform: capitalize;
+  }
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_cards.scss:109]: Add a blank line here, please

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.form-actions-comment {
+  .btn {
+    background-color: $links;
+    font-weight: 700;
+    margin-bottom: 1.3rem;
+  }
+
+  .btn:hover {
+    background-color: #caa006;
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:47]: Use a variable to add this color

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.form__button--back {
+  background-color: $red;
+  margin-left: 2rem;
+  margin: 2rem;
+  max-width: 100px;
+  padding: 0.5rem 0.5rem;
+  text-align: center;
+
+  a {
+    color: white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:61]: Use a variable to add this color

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    text-decoration: none;
+  }
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:64]: Add a blank line here please

**Review by JuanequeX — CHANGES_REQUESTED** [review]
`_post_comment.sccs` is the same file whit a same code that you have in another PR, be carefull whit this because can be generate conflicts, i will recommend that wait to merge first PR to upload another one after.

**Review by YilKanda — COMMENTED** [review]
Please solve the suggestions made by the mentor @JuanequeX

**Review by juperameza — DISMISSED** [review]
LGTM!!!

File: app/views/orders/show.html.erb [hunk 5]
```diff
@@ -1,7 +1,13 @@
...
+		<div class="card__order card">
+			<% @order.order_details.each do |d|%>
+				<h2><%= d.post.title%> </h2>
+				<p>Cantidad: <%= d.quantity%></p>
+			<% end %>
+			<p>Comprado el: <%= @order.created_at.strftime("%d de %B del %Y a las %H:%M") %></p>
+			<h4>Importe total: <span>$<%= @order.total%></span></h4>
+		</div>
+	</div>
+</div>
```
Comment (YilKanda) [inline app/views/orders/show.html.erb:13]: Check the indentation of your file

**Review by germanruzca — DISMISSED** [review]

File: app/assets/stylesheets/components/_cards.scss [hunk 6]
```diff
@@ -81,9 +81,29 @@
...
-  i {
+  a,i {
     color: $text-gray;
     text-decoration: none;
   }
 }
+
+.star {
+  color: $yellow;
+}
```
Comment (EliasUR) [inline app/assets/stylesheets/components/_cards.scss:92]: Delete this, we are already using this class in another file

**Review by EliasUR — CHANGES_REQUESTED** [review]

**Review by JuanequeX — DISMISSED** [review]
LGTM

**Review by germanruzca — DISMISSED** [review]

**Review by EliasUR — DISMISSED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]
LGTM!

**Review by juperameza — APPROVED** [review]
LGTM!!!

**Review by Javi1499 — APPROVED** [review]

**Pull request merged**